astronomy/trappist_dynamics_test.cpp

```diff
@@ -1049,7 +1049,8 @@ TEST_F(TrappistDynamicsTest, DISABLED_Optimisation) {
   SolarSystem<Sky> const system(
       SOLUTION_DIR / "astronomy" / "trappist_gravity_model.proto.txt",
       SOLUTION_DIR / "astronomy" /
-          "trappist_initial_state_jd_2457000_000000000.proto.txt");
+          "trappist_preoptimisation_initial_state_jd_2457000_000000000.proto."
+          "txt");
```

Comment:
Fold the line *before* one of the `.`, `.txt` or `.proto.txt` are recognizable as extensions, whereas a full-stop terminated filename looks weird.

Reply:
Done.